tex/future_extensions.tex

```diff
@@ -1 +1,22 @@
-% TODO: FUTURE EXTENSIONS
\ No newline at end of file
+\subsection{Machine Learning}
+
+Up till this point, only a very standard edge matching method is used and the process is purely mathematical. This exports some rather serious problems:
```

Comment:
'The current implementation uses a standard edge matching method, with the matching process remaining purely mathematical. Such a limitation exposes some rather serious problems:'

Would be better?